runtime/stdlib/contracts/test.cdc

```diff
@@ -312,6 +319,11 @@ pub contract Test {
         /// Resets the state of the blockchain to the given height.
         ///
         pub fun reset(to height: UInt64)
+
+        /// Moves the time of the blockchain by the given delta,
+        /// which should be passed in the form of seconds.
+        ///
+        pub fun moveTime(by delta: UFix64)
```

Comment:
Seems in the implementation, we are trimming off the decimal points. So maybe it would make sense to just use `UInt64` here?

Reply:
This is probably for consistency with [`Block.timestamp`](https://github.com/onflow/cadence/blob/e287ab0316547b0a7bd6a527eb9b67115d6dc834/runtime/sema/block.cdc#L26C13-L26C30)

Reply:
oh, that makes sense 👍

Reply:
Yes, exactly, just like Bastian mentioned.